docs/development/releasing.rst

```diff
@@ -442,6 +460,82 @@ be fixed soon to a new release milestone.  If the upcoming bug fix release is
 issues that you don't expect to be fixed in time for 'v0.2.2'.
 
 
+.. _helpers-release-info:
+
+Coordinating Astropy and astropy-helpers Releases
+-------------------------------------------------
+
+A bit more initial effort is required for an Astropy release that has a
+corresponding astropy-helpers_ release.  The main reason for this more complex
+procedure is to allow the Astropy core to be tested against the new helpers
+before anything is released.  Hence the following procedure should be added
+to the beginning of the above procedure when this is required. This procedure
+applies both for regular release *and* release candidates are the same
+(except that version numbers have ``rc#`` at the end).
+
+#. In the `astropy-helpers repository`_, create a new (temporary) branch
+   "tmp-release-v<version>"::
+
+      $ cd /wherever/you/put/astropy/astropy_helpers
+      $ git branch tmp-release-v<version> <maintenance branch name>
```

Comment:
(Note to myself from an out-of-band conversation with @astrofrog: instead of a local version, just do ``git push <maintenance branch name>:tmp-release-v<version>``)